src/components/dashjs-statistics/dashjs-statistics.tsx

```diff
@@ -1,296 +1,416 @@
-import { Component, Host, h, Prop, Watch, Element, State} from '@stencil/core';
+import { Component, Host, h, Prop, Watch, Element, State, Listen } from '@stencil/core';
 import * as chartjs from 'chart.js';
 const { Chart } = chartjs.default.Chart;
 
-
 @Component({
   tag: 'dashjs-statistics',
   styleUrl: 'dashjs-statistics.css',
   shadow: false,
 })
-
 export class DashjsStatistics {
+  @Prop()
+  video_data: any;
+  @Prop()
+  audio_data: any;
+
+  @State()
+  videoDisable: string = 'Disable';
```

Comment:
please use booleans

Reply:
@hieu2010 should I change these to booleans?

Reply:
yes